doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1005,7 +1005,162 @@ Response code:
+* 'notification' : It is an object that describes the notification received by the subscriber.
+* 'options' : It is a dictionary which can contain different parameters related to the subscription, namely the duration.
+
+A 'subject' contains the following subfields:
+
+* 'entity' : Id or pattern id which identifies the affected entities (optional)
+* 'type' : Type of the affected entities (optional)
+* 'attributes': Attributes to be monitored (if not specified it would mean any)
+* 'condition': Condition that will trigger the notification. It could be 'ONCHANGE' or 'ONTIMEINTERVAL'.
+By default will be ONTIMEINTERVAL. Please note that at the time of writing the editors are considering to combine
+attributes and condition to allow expressions like 'temperature > 25.5'. So this is likely to change.
+
+Entity or type must be present.
+
+A 'notification' object contains the following subfields:
+
+* 'attributes': Attributes to be included in the notification message. If not specified they will be the same as those which are monitored
+* 'callback' : URL that will point to the service which will be invoked when a notification is generated
+
+### List subscriptions [GET /v2/subscriptions]
+
+Returns a list of all the subscriptions present in the system
+
++ Response 200
+
+        [
+            {
+                "id": "abcdefg",
+                "subject": {
+                    "entity": "Bcn_Welt",
+                    "type": "Room",
+                    "attributes": [
+                        "temperature"
+                    ],
+                    "condition": "ONCHANGE"
+                },
+                "notification": {
+                    "callback": "http://localhost:1234",
+                    "attributes": ["temperature", "humidity"],
+                    "throttling": "PT5S"
+                },
+                "options": {
+                    "duration": "1M"
+                }
+            }
+        ]
+
+### Create a new subscription [POST /v2/subscriptions]
+
+Creates a new subscription. The subscription is represented by a JSON object as described at the beginning of this section.
+
++ Request (application/json)
+
+        {
+            "subject": {
+                "type": "Room",
+                "attributes": [
+                      "humidity"
+                ],
+                "condition": "ONTIMEINTERVAL"
+            },
+            "notification": {
+                "callback": "http://localhost:1234",
+                "attributes": ["temperature", "humidity"],
+                "throttling": "PT5S"
+            },
+            "options": {
+                "duration": "1M"
+            }
+        }
+
++ Response 201
+
+    + Headers
+
+            Location: /v2/subscriptions/abcde98765
+
+## Context Providers [/v2/providers]
+
+This section is still under discussion.
+
+A context provider is represented by a JSON object with the following fields:
+
+* 'id': Unique identifier assigned to the provider
```

Comment:
Not exactly... it would be de id associated to the registrations. Note that different registrations can use the same context provider (in the providingApplication/provider field).